src/log.c

```diff
@@ -232,8 +232,8 @@ log_level_from_string(char *log_level)
         return PROF_LEVEL_WARN;
     } else if (strcmp(log_level, "ERROR") == 0) {
         return PROF_LEVEL_ERROR;
-    } else { // default to info
-        return PROF_LEVEL_INFO;
+    } else { // default logging is warn
+        return PROF_LEVEL_WARN;
```

Comment:
shouldnt this one stay info?

Reply:
When I understand well, it's just called from main during init of profanity.

main.c
prof_run(log ? log : "INFO", account_name, config_file, log_file, theme_name);

prof_run calling: _init(log_level, config_file, log_file, theme_name)

profanity.c:_init(char *log_level, char *config_file, char *log_file, char *theme_name)
log_level_t prof_log_level = log_level_from_string(log_level);

if we would use NULL or "WHATEVER" it will still use the WARN level as default.